Help.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+#include "Help.h"
+#include "Menu.h"
+
+ 
+using namespace std;
+
+
+    void Help::display() 
+{

```

Comment:
Too long. Single responsibility principle is not respected here.

Reply:
How ? it just displays the help paragraph. Is it because of the request to go bak to menu ?

Reply:
First, this should return a string. So it can be tested. Second, as I explained previously the main function should handle displaying (calling std::cout).

Reply:
ok i understand. So I should change all void methods (in every class). 
But i dont understand how it doesnt respect the single responsibility principle since it does one thing which is displaying.

Reply:
If it does only displaying why there are:
1. `cin >> res;` (reading input from the keyboard)
2. `system("clear");` (clearing output)